types/upgrade.go

```diff
@@ -25,6 +25,15 @@ const (
 	FixDoubleSignChainId        = "FixDoubleSignChainId"
 	BEP126                      = "BEP126" //https://github.com/binance-chain/BEPs/pull/126
 	BEP255                      = "BEP255" // https://github.com/bnb-chain/BEPs/pull/255
+
+	// TODO: to be determined
+	BCFusionFirstHardFork  = "BCFusionFirstHardFork"
```

Comment:
Let us rename to `FirstSunsetFork`, `SecondSunsetFork` and ``FinalSunsetFork``

Reply:
fixed